packages/web3-eth-contract/package.json

```diff
@@ -22,7 +22,10 @@
 		"test:ci": "jest --coverage=true --coverage-reporters=json --verbose",
 		"test:watch": "npm test -- --watch",
 		"test:unit": "jest --config=./test/unit/jest.config.js",
-		"test:integration": "jest --config=./test/integration/jest.config.js --runInBand --forceExit"
+		"test:integration": "jest --config=./test/integration/jest.config.js --runInBand --forceExit",
```

Comment:
Wasn't `--forceExit` what was hiding the uncaught rejection error?

Reply:
It was mixed case, if the unhandled rejection is the last test then it just exit because of this flag. If the unhandled rejection appears in some other tests, it caused that test to fail.